### ISSUE TITLE
Brainstorming the tutorial

### ISSUE THREAD

**Issue by budschi** [FAIRmat-NFDI/AreaA-Documentation#1]
defining content, structure, responsibilities, deadlines etc in [brainstorming_tutorial.md](https://github.com/FAIRmat-NFDI/AreaA-Documentation/blob/main/tutorial/brainstorming_tutorial.md)

**Comment by budschi** [comment]
Please add your thoughts here or in the md file

**Comment by budschi** [comment]
# Conclusion from our meeting 8.02.2023:

## Content of tutorial:

1. introdution + definitions --> Andrea @aalbino2 
2. NOMAD usage #3 --> Florian @domna 
3. data revision + write schema --> Hampus @hampusnasstrom 
4. more advanced usage:
	* xlsx + plotting --> Andrea @aalbino2 
	* base classes + referencing --> Hampus @hampusnasstrom 
	
5. explore data --> Pepe @Pepe-Marquez 
	table with chemicals cas call

## general information
* date: 14:00 to 17:00 (break at ~15:30)
* title: Electronic lab notebook (ELN) functionality in NOMAD 
* keywords: structured data, ELN, NOMAD, schema, ...
* tutorial style: pingpong of lecture and hands-on part, from simple to more complex content
* use [tutorial folder](https://github.com/FAIRmat-NFDI/AreaA-Documentation/blob/main/tutorial/) as working folder

## next actions

- [ ] everyone prepares an outline of his part based on [brainstorming_tutorial.md](https://github.com/FAIRmat-NFDI/AreaA-Documentation/blob/main/tutorial/brainstorming_tutorial.md) 

- [ ] outline of sections will be discussed in next TF meeting on 13.02.2023 13:30h

**Comment by domna** [comment]
# Ideas for the `NOMAD Usage` part

- Walk through the different tabs of nomad and show what users can do there (Publish, Explore, Analyze). ~ 5min
- Show the workflow of creating and publishing an upload (ideally with an eln example) ~ 5min
   - Explain embargo
   - What happens when the upload is published? Maybe we can show a published upload and revoke it afterwards?
   - Maybe nice to already use a schema which Hampus will build in the next section
   - Quickly show parsed parts (but mainly refer to in-depth discussions later in the tutorial)
- Interaction with north, show a small one of the available examples (e.g. ellipsometry, xps) ~5min

This are some points I can think of. We can also deepen certain points and leave others out (Especially, I'm not sure whether to open the north topic here or just stay with the eln completely - but from my experience people find this integration nice and quickly see the benefit of using sth like nomad). Times are the minimum, so I think this can be adapted to also take more time if needed.

Further discussions in #3

**Comment by andreaa93** [comment]
General questions:

* which Nomad to use for demo? 
* How to deal with "data" files: upload them with "schema" files or create them with clicks from the GUI?

**Comment by domna** [comment]
> General questions:
> 
> * which Nomad to use for demo?

We could use the nomad-lab/util deployment. north is working there and we can select the branch we wan't to deploy there. It is currently being used for deploying the nexus search branch but I think we probably can use it in march.

**Comment by Pepe-Marquez** [comment]
> General questions:
> 
> * which Nomad to use for demo?
> * How to deal with "data" files: upload them with "schema" files or create them with clicks from the GUI?

I am against adding data with the schema. As a matter of fact, doing something like this was breaking one of my schemas because the entry type "Schema" was not detected properly.

**Comment by andreaa93** [comment]
> > General questions:
> > 
> > * which Nomad to use for demo?
> > * How to deal with "data" files: upload them with "schema" files or create them with clicks from the GUI?
> 
> I am against adding data with the schema. As a matter of fact, doing something like this was breaking one of my schemas because the entry type "Schema" was not detected properly.

Yes, definitely data and definitions must not be in the same file.

The further thing I meant is: do we show a data file, or we just instantiate it within the guy with the "create new entry" button? Let's see which method is the most simple

**Comment by budschi** [comment]
> > General questions:
> > 
> > * which Nomad to use for demo?
> 
> We could use the nomad-lab/util deployment. north is working there and we can select the branch we wan't to deploy there. It is currently being used for deploying the nexus search branch but I think we probably can use it in march.

We should use an official NOMAD version, i.e. the beta version.  What is missing in the current beta and what will be merged in soon? @markus1978 told me Jupyter Hub will be in the public beta soon.
